UnoCheck/Solutions/GitCliSolution.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DotNetCheck.Models;
+using Microsoft.Win32;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class GitCliSolution : Solution
+	{
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			await base.Implement(sharedState, cancellationToken);
+
+			var ps = new ProcessStartInfo(WinCliUrl)
+			{
+				UseShellExecute = true,
+				Verb = "open"
+			};
+			_ = Process.Start(ps);
+
+			ReportStatus("The Phyton version can be downloaded from Windows Store.");
```

Comment:
This needs to be updated. If running on Windows, you can also suggest to use the visual studio installer.

Reply:
You were right! Done!